datajunction-server/datajunction_server/internal/nodes.py

```diff
@@ -1388,6 +1389,29 @@ async def column_lineage(
     return lineage_column
 
 
+async def derive_sql_column(
+    cube_element: CubeElementMetadata,
+) -> ColumnOutput:
+    """
+    Derives the column name in the generated Cube SQL based on the CubeElement
+    """
+    query_column_name = None
+    if cube_element.type == "metric":
+        query_column_name = cube_element.name
+    elif cube_element.type == "dimension":
+        query_column_name = (
+            f"{cube_element.node_name}{SEPARATOR}{cube_element.name}".replace(
+                SEPARATOR,
+                "_DOT_",
```

Comment:
Can this part be extracted out into a constant (or actually I think it's already a constant) so that if we do decide to change this value, we can just change the constant?

Reply:
I forgot we have this `amenable_name` function that will take care of this entirely so I switched to using that.